index.html

```diff
@@ -1,258 +1,320 @@
 <html lang="en">
-<head>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <meta
+      name="description"
+      content="Cindy Kandie is a well equipped frontend developer who has taken time to polish her craft. This Portfolio is a visual representation of the work she has done and what she can do."
+    />
+    <meta
+      name="keywords"
+      content="frontend developer, web developer"
+    />
 
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="Cindy Kandie is a well equipped frontend developer who has taken time to polish her craft. This Portfolio is a visual representation of the work she has done and what she can do." />
-    <meta name="keywords" content="frontend developer, web developer" />
-    
-    <link rel="stylesheet" href="css/styles.css">
-    <link rel="stylesheet" href="css/footer.css">
-    <link rel="stylesheet" href="css/mediascreen.css">
+    <link rel="stylesheet" href="css/styles.css" />
+    <link rel="stylesheet" href="css/footer.css" />
+    <link rel="stylesheet" href="css/mediascreen.css" />
 
-    <link rel="shortcut icon" href="assets/favicon.ico" type="image/x-icon">
+    <link
+      rel="shortcut icon"
+      href="assets/favicon.ico"
+      type="image/x-icon"
+    />
 
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=BhuTuka+Expanded+One&family=Poppins:wght@200;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=BhuTuka+Expanded+One&family=Poppins:wght@200;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
 
-    <title>Cindy Kandie's Web Development Portfolio</title>
-</head>
-<body>
-    <header id= "item-0">
-        <div class="logo">
-            <p>Cindy K</p>
-        </div>
-        <nav>
-            <img src="assets/menu.png" alt="navbar">
-            <ul class="desktop-nav">
-                <li> <a href="">Portfolio</a></li>
-                <li> <a href="">About</a></li>
-                <li> <a href="">Contact</a></li>
-            </ul>
-        </nav>
+    <title>Cindy Kandie - Cindy's Web Development Portfolio</title>
+  </head>
+  <body>
+    <header id="item-0">
+      <div class="logo">
+        <p>Cindy K</p>
+      </div>
+      <nav>
+        <img src="assets/menu.png" alt="menu bars" />
```

Comment:
- [ ] Choosing a descriptive and better-representing alt text is useful for interacting and understanding the content if the image is not important 👊🏽.